crates/sdk/src/tx.rs

```diff
@@ -3778,6 +3783,7 @@ pub async fn gen_ibc_shielding_transfer<N: Namada>(
             // Fees are paid from the transparent balance of the relayer
             None,
             true,
+            args.expiration.to_datetime(),
```

Comment:
@yito88 we might have a failure in Hermes because of this update...

Reply:
Tests with Hermes worked well

Reply:
Right seems like everything is fine now, thanks!